### PR TITLE
Add test plan for thermal control feature

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@stepanblyschak please review as well

**Comment by keboliu** [comment]
@sujinmkang would you please help to review

File: doc/pmon/sonic_thermal_control_test_plan.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
+# Thermal Control Test Plan
+
+## Table of Content
```
Comment (jleveque) [inline doc/pmon/sonic_thermal_control_test_plan.md]: "Contents"

Comment (Junchao-Mellanox) [reply doc/pmon/sonic_thermal_control_test_plan.md]: Fixed

**Pull request merged**